erpnext/manufacturing/doctype/routing/routing.py

```diff
@@ -11,6 +11,15 @@
 class Routing(Document):
 	def validate(self):
 		self.set_routing_id()
+		self.update_op_time()
+
+	def update_op_time(self):
```

Comment:
```suggestion
	def update_operation_time_in_linked_BOMs(self):
```

This or something shorter that conveys what function does.